src/components/loadButton.js

```diff
@@ -0,0 +1,4 @@
+const createBtnLoadMore = () =>
```

Comment:
Переименуй, по аналогии с другими компонентами 
```suggestion
const createLoadButtonElement = () =>
```